server/server.go

```diff
@@ -179,6 +179,11 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 			}
 		}
 		if userConfig.BitbucketUser != "" {
+			// The default BitbucketBaseURL is https://api.bitbucket.org which can't actually be used for git
+			// so we overide it here only if it's that to be bitbucket.org
+			if userConfig.BitbucketBaseURL == "https://api.bitbucket.org" {
+				userConfig.BitbucketBaseURL = "bitbucket.org"
```

Comment:
I think this might break everything else that relies on that config. What about
```go
			// The default BitbucketBaseURL is https://api.bitbucket.org which can't actually be used for git
			// so we override it here only if it's that to be bitbucket.org
                        bitbucketBaseURL := userConfig.BitbucketBaseURL
			if bitbucketBaseURL == "https://api.bitbucket.org" {
				bitbucketBaseURL = "bitbucket.org"
			}
			if err := events.WriteGitCreds(userConfig.BitbucketUser, userConfig.BitbucketToken, bitbucketBaseURL, home, logger); err != nil {
				return nil, err
			}
```

Reply:
Yeah. Good point I think you're right I didn't think about that. I'll fix that up now.